Soundify/Soundify/Data/Model/Album.swift

```diff
@@ -22,6 +22,7 @@ struct Album: BaseModel {
     var id: String = ""
     var images: [Image] = []
     var name: String = ""
+    var releaseDate: Date = Date()
```

Comment:
```suggestion
    var releaseDate: Date?
```

riêng Date không sử dụng giá trị mặc định, vì nếu trong trường hợp API ko trả về dữ liệu nó sẽ default hiển thị ngày giờ hiện tại.